mobile/native/src/ln_dlc/mod.rs

```diff
@@ -356,7 +356,7 @@ async fn keep_wallet_balance_and_history_up_to_date(node: &Node) -> Result<()> {
 }
 
 pub fn get_new_address() -> Result<String> {
```

Comment:
🤔 Is this returning a result because of a trait, or could we make that infailable?

Reply:
Good catch!